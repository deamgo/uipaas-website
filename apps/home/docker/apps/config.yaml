database:
  host: uipaashome-mysql
  port: 3306
  user: root
  password: "rootroot"
  dbname: uipaas-home
  charset: utf8mb4