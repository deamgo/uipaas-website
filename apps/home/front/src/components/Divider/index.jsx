import './index.less'

export const Divider = ({margin}) => {
    return (
        <div className="divider" style={margin}></div>
    )
}
