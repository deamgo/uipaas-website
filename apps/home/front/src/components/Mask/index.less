.mask {
  position: fixed;
  left: 0;
  top: 0;
  width: 1920rem;
  height: 120vh;
  opacity: 1;

  background-color: rgba(0, 0, 0, 0.7);
  z-index: 1005;
}