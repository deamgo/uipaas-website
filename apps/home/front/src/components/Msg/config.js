export const MessageApi = {
    info: (text) => {},
    success: (text) => {},
    warning: (text) => {},
    error: (text) => {},
  };
  
  export const List = {
    text: '',
    key: '',
    type: 'info',
  };
  