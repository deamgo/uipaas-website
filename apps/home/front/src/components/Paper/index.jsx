import './index.less'

export function Paper({ title, content, ps }) {
    return (
      <div className="ctr" style={ps}>
        <span className="p__title">{title}</span>
        <br />
        <span className="p__content">{content}</span>
      </div>
    );
  } 