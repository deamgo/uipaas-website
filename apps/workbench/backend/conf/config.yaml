database:
  host: 127.0.0.1
  port: 3306
  user: root
  password: 12345678
  dbname: workbench
  charset: utf8mb4

  redis:
    addr: 127.0.0.1:6379
    password:
    db: 1
