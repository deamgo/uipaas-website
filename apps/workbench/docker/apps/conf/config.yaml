database:
  host: workbench-mysql
  port: 3306
  user: root
  password: rootroot
  dbname: workbench
  charset: utf8mb4

  redis:
    addr: 127.0.0.1:6379
    password:
    db: 1
email_config:
  path: /home/workbench/backend/mjml
permission_config:
  path: /home/workbench/backend/auth/permission/permissions.csv
