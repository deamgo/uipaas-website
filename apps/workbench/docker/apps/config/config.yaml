database:
  host: workbench-mysql
  port: 3306
  user: root
  password: rootroot
  dbname: workbench
  charset: utf8mb4
