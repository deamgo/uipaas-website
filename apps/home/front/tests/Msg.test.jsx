// index.test.js
import { render, act, getByTitle } from "@testing-library/react";
import "@testing-library/jest-dom";
import { MessageContainer } from "../src/components/Msg";
import $message from "../src/components/Msg";
import { MessageApi } from "../src/components/Msg/config";
import Msg from "../src/components/Msg/Msg";

test("renders messages correctly", () => {
  const { getByText } = render(<MessageContainer />);

  //   $message.info("This is an info message");

  //   expect(getByTitle("This is an info message")).toBeInTheDocument();
  expect($message.info("This is an info message"));
});

test("renders messages correctly", () => {
  const { getByText } = render(<MessageContainer />);
});

test("handles multiple messages correctly", () => {
  const { getByText } = render(<MessageContainer />);

  expect($message.success("Success message"));
  expect($message.warning("Warning message"));
  expect($message.error("Error message"));
});

test("message api", () => {
  expect(MessageApi.info())
  expect(MessageApi.success())
  expect(MessageApi.error())
  expect(MessageApi.warning())
})

describe('Msg', () => {
  it('renders correctly', () => {
    const { container } = render(<Msg type="success" text="This is a success message." />);
    expect(container.firstChild).toMatchSnapshot();
  });
});