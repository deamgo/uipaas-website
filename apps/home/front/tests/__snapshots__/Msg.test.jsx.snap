// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Msg > renders correctly 1`] = `
<div
  class="message success"
>
  <span
    class="icon"
  />
  <span>
    This is a success message.
  </span>
</div>
`;

exports[`Msg renders correctly 1`] = `
<div
  class="message success"
>
  <span
    class="icon"
  />
  <span>
    This is a success message.
  </span>
</div>
`;
